troposphere/static/js/components/admin/ImageRequest.react.js

```diff
@@ -110,7 +110,7 @@ define(function (require) {
               onChange={this.handleResponseChange}/><br />
             <button disabled={request.get('status').name != 'pending'}onClick={this.approve} type="button" className="btn btn-default btn-sm">Approve</button>
             <button disabled={request.get('status').name != 'pending'}onClick={this.deny} type="button" className="btn btn-default btn-sm">Deny</button>
-            <button disabled={request.get('status').name == 'rejected'}onClick={this.resubmit} type="button" className="btn btn-default btn-sm">Re-Submit</button>
+            <button disabled={request.get('status').name == 'closed'}onClick={this.resubmit} type="button" className="btn btn-default btn-sm">Re-Submit</button>
```

Comment:
At a later date 'how we disable things' may change, and it'd be good to get this into a function rather than inline. For now though, :+1: